lib/screens/settings/quick_links.dart

```diff
@@ -109,7 +110,13 @@ class _QuickLinksScreenState extends State<QuickLinksScreen> {
                     textInputAction: TextInputAction.done,
                     onEditingComplete: addLink),
               ),
-              IconButton(icon: const Icon(Icons.add), onPressed: addLink),
+              ValueListenableBuilder<bool>(
```

Comment:
Hmm it might be worthwhile to extract this into a new component or call `setState()` here instead. Since the component is already stateful it's a bit odd to have another listener inside of it.

Reply:
I think setState requires a rerender of all the widget and ValueNotifier does not, I will look into extracting into a separate widget.